app/src/main/assets/DOC_HTML/apps/P_CapacitivePhaseShift.md

```diff
@@ -1,6 +1,8 @@
-CR Phase Shift
+R-C Phase Shift
 ---
 	
+#### Schematics
```

Comment:
In one place you have space between and in other place don't have. Make it all uniform